homeassistant/components/media_player/apple_tv.py

```diff
@@ -132,6 +146,18 @@ def _has_playing_media_changed(self, new_playing):
         return new_playing.media_type != old_playing.media_type or \
             new_playing.title != old_playing.title
 
+    def turn_on(self):
+        """Turn the media player on."""
+        self._is_off = False
+        self._reset()
+        self.schedule_update_ha_state()
```

Comment:
There is no need to call this. Your device does not overwrite `should_poll` to return `False`, so Home Assistant is polling this device, including polling it right after it calls a service.